clar2wasm/tests/wasm-generation/comparison.rs

```diff
@@ -0,0 +1,33 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use clarity::vm::Value;
+use proptest::proptest;
+
+use crate::string_ascii;
+
+fn filter_invalid_char(input: &Value) -> Value {
+    let invalid_chars: [char; 1] = ['\''];
+    let v: String = input.clone().expect_ascii().replace(invalid_chars, "");
+
+    Value::Sequence(clarity::vm::types::SequenceData::String(
+        clarity::vm::types::CharType::ASCII(clarity::vm::types::ASCIIData { data: v.into() }),
+    ))
+}
+
+const COMPARISONS_FUNC: [&str; 4] = ["<", "<=", ">", ">="];
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_comparison_string_ascii(val1 in string_ascii(50u32), val2 in string_ascii(50u32)) {
```

Comment:
Why the choice of only ascii strings?
A single test could be used for all supported types if you put them all in a `prop_oneof!`.

Reply:
I think that using `prop_oneof!` to functions that needs more than one input will test several cases like below, where the input types are different (and throws an error):

```
(< 3 u5)
(>= 0x98 "abcde")
(<= u24323 0x8777)
```

We can try to filter the inputs to be the same type, but that will add complexity to tests. Probably is more straightforward to add tests for each types.

Reply:
Ok... let me check that approach.